src/main/java/org/tinyradius/util/RadiusServer.java

```diff
@@ -287,11 +289,11 @@ public void setDuplicateInterval(long duplicateInterval) {
 	}
 
 	/**
-	* Returns a list containing received packets
+	* Returns a map containing received packets
 	*
 	* @return list of received packets
 	*/
-	public List getReceivedPackets() {
```

Comment:
Thanks for the change.  I'm wondering if we need to retain backward compatibility since this is a public method?   Otherwise, we should bump the pom version.

Reply:
This method was introduced in pull request https://github.com/ctran/TinyRadius/pull/31 which was not released so I think it is ok to release in as is (bump pom version) as it will not effect backward compatibility